.github/workflows/release.yml

```diff
@@ -28,3 +28,11 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npm run semantic-release
+      - name: Build documentation
+        run: npm install && npm run docs
```

Comment:
No need to install again, no ?

Reply:
there is, for some reasons it does fail without that install - saying that there is no npm_modules

Reply:
i would rather do `npm i` in `package.json` for `docs` script, than here... otherwise building docs will fail in console in some cases for same reason as it was failing here